resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorPodsAllocator.scala

```diff
@@ -339,6 +339,9 @@ private[spark] class ExecutorPodsAllocator(
         resources
           .filter(_.getKind == "PersistentVolumeClaim")
           .foreach { resource =>
+            if (conf.get(KUBERNETES_DRIVER_OWN_PVC) && driverPod.nonEmpty) {
+              addOwnerReference(driverPod.get, Seq(resource))
+            }
```

Comment:
This overrides the added owner reference at L338 `addOwnerReference(createdExecutorPod, resources)`, right?

Reply:
Yes, correct!